app.js

```diff
@@ -0,0 +1,280 @@
+// DOM API를 사용하기 위해 해당 객체들을 변수로 가져온다
+const todoListAreaDiv = document
+  .querySelector('.todo-area')
+  .querySelector('.showing-list-area');
+
+const doneListAreaDiv = document
+  .querySelector('.done-area')
+  .querySelector('.showing-list-area');
+
+const inputBox = document.querySelector('.todo-input');
+const submitButton = document.querySelector('.submit-button');
+
+const todayDateInput = document.querySelector('.date-input');
+const currentDate = new Date(); // 현재의 날짜에 해당함
+const formattedDate = currentDate.toISOString().slice(0, 10); // 현재의 날짜를 YYYY-MM-DD 포맷에 맞추어 문자열로 변경함
+todayDateInput.value = formattedDate;
+
+// 현재 선택된 날짜의 YYYY-MM-DD 문자열을 반환하는 함수
+function getNowDateFormatString() {
+  return todayDateInput.value;
+}
+
+function removeAllChildNodes(parent) {
+  while (parent.firstChild) {
+    parent.removeChild(parent.firstChild);
+  }
+}
+
+function makeUniqueTodoIDByDate() {
+  const now = Date.now();
+  const stringNow = now.toString();
+  return stringNow;
+}
+
+makeUniqueTodoIDByDate();
+
+// 모든 todo리스트를 지우고 로컬 스토리지에 업데이트된 것으로 다시 마운트하는 함수
+function getDownAllTodoItemsFromDomAndRemount() {
+  removeAllChildNodes(todoListAreaDiv);
+  let newTodoList =
+    localStorage.getItem(`${todayDateInput.value}todo`) === null
+      ? []
+      : JSON.parse(localStorage.getItem(`${todayDateInput.value}todo`));
```

Comment:
이 부분 로직이 조금 복잡한 것 같아서 이런식으로 수정해보시는건 어떨까요?
```suggestion
function getDownAllTodoItemsFromDomAndRemount() {
   removeAllChildNodes(todoListAreaDiv);
   const newTodoList = JSON.parse(localStorage.getItem(`${todayDateInput.value}todo`) || '[]');
```

Reply:
함수명도 `removeTodoItems` 와 같이 간단하게 하면 좋을 것 같습니다